textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/sakai/SakaiFsService.java

```diff
@@ -167,6 +167,11 @@ public String getVolumeId(FsVolume volume) {
 		List<FsVolume> volumes = new ArrayList<>(sites.size());
 		for (Site site: sites) {
 			String currentSiteId = site.getId();
+			//SAK-30922 - Do not search admin site for tools
```

Comment:
I know this is picky but we don't need the JIRA issues as comments. Most editors you just click show annotations and it shows the blame info for every line so there is no need to leave comments like this...